### PR TITLE
Add GL_EXT_maximal_reconvergence

### PR DESCRIPTION
**Pull request by alan-baker**

### CODE REVIEW

**Comment by alan-baker** [comment]
@gnl21 PTAL

File: extensions/ext/GL_EXT_maximal_reconvergence.txt [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  Specification, version 4.60 (revision 7), dated July 10, 2019.
+
+  Interacts with GL_KHR_vulkan_glsl.
+
+  Interacts with GL_EXT_control_flow_attributes.
+
+Overview
+
+  This extension adds the ability to annotate entry points via an attribute to
+  indicate that subgroups should reconverge in the same way as invocation
```
Comment (jeffbolznv) [inline extensions/ext/GL_EXT_maximal_reconvergence.txt:48]: Is this leftover from subgroup_uniform_control_flow?

File: extensions/ext/GL_EXT_maximal_reconvergence.txt [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  groups.
+
+  This is done through the attribute syntax "[[ ... ]]" syntax. For example,
+    
+    void main() [[maximally_reconverges]] { ...  }
```
Comment (jeffbolznv) [inline extensions/ext/GL_EXT_maximal_reconvergence.txt:53]: I just noticed I spelled this differently in the glslang code I pushed this morning, `[[maximal_reconvergence]]`. I think I may have written that code before the actual spec was drafted. Should I change glslang or do you want to change the spec?

Comment (alan-baker) [reply extensions/ext/GL_EXT_maximal_reconvergence.txt:53]: I think I used `[[maximally_reconverges]]` in the blog post so a slight preference to update glslang, but we could also update the blog.

File: extensions/ext/GL_EXT_maximal_reconvergence.txt (hunk 1 above)
Comment (alan-baker) [reply extensions/ext/GL_EXT_maximal_reconvergence.txt:48]: I think so.

File: extensions/ext/GL_EXT_maximal_reconvergence.txt (hunk 2 above)
Comment (jeffbolznv) [reply extensions/ext/GL_EXT_maximal_reconvergence.txt:53]: OK, I'll fix glslang.

Comment (jeffbolznv) [reply extensions/ext/GL_EXT_maximal_reconvergence.txt:53]: This fix was merged: https://github.com/KhronosGroup/glslang/pull/3489

**Review by jeffbolznv — APPROVED** [review]

**Comment by gnl21** [comment]
LGTM. Is this ready to merge?

**Comment by alan-baker** [comment]
I think so.

**Pull request merged**